_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -2217,11 +2218,6 @@ inputs = {
 
   # Set to point to the AWS Account that has set up the Security Hub organisation and has sent out invitations to other AWS Child accounts. For example, get the `root` account to send invitations to `security`, `logs`, `dev`, etc.
   security_hub_associate_to_master_account_id = "<ROOT_ACCOUNT_ID>"
-  security_hub_external_member_accounts = { security = {
-                                               account_id = "<LOGS_ACCOUNT_ID>"
-                                               email      = "root-accounts+logs@acme.com"
-                                             }
-                                        }
```

Comment:
I'm wondering why this is removed? Is it no longer necessary?

Reply:
Because this is set in the root account. This account only needs to add the root account id.

Reply:
@ina-stoyanova @marinalimeira let me know if we can resolve this one?